src/searches/Lines/Lines.js

```diff
@@ -10,7 +15,9 @@ const lineMeasuresRegExp = new RegExp(
   '([0-9]*)\\s*([0-9]+\\.?[0-9]*)\\-([0-9]*\\.?[0-9]*)',
 );
 
-const lineKilometerRegExp = new RegExp('^([0-9]+)\\s+([0-9]+\\.?[0-9]+)$');
+const lineKilometerRegExp = new RegExp(
+  '^([0-9]+)(\\s)(\\+)?(\\s+)?([0-9]+(\\.[0-9]+)?)$',
+);
```

Comment:
I think there is  test somewhere for this regex, can you check and if not can you add tests for the new possible values.pls

Reply:
I didn't find any present tests, so I added some in Lines.test.js